test/helper/e2e/k8s/operator.go

```diff
@@ -72,7 +72,7 @@ func BuildManager(initCfg *Config) (manager.Manager, error) {
 	logger.Info("starting manager", zap.Any("config", config))
 
 	var cacheFunc cache.NewCacheFunc
-	if len(config.WatchedNamespaces) > 1 {
+	if len(config.WatchedNamespaces) > 0 {
```

Comment:
Same error was duplicated in e2e tests. I think this was done on purpose and made sense probably on the previous version of controller runtime, before the manual migration that brought up the bug.